### PR TITLE
Add instructions to configure RDS instances for Off-Host Monitoring

### PR DESCRIPTION
**Pull request by potocnyj**
Ref #64 https://github.com/VividCortex/docs/issues/36

### CODE REVIEW

**Comment by potocnyj** [comment]
@Andertech If you have any suggestions on formatting I'd appreciate it :+1: 
@Preetam mind reviewing the content for me?

File: content/getting-started/installing.md [hunk 1]
```diff
@@ -79,11 +79,30 @@ the `PERFORMANCE_SCHEMA` tables. The `statements_digest` consumer needs to be en
...
 privileges to see the query text for other users.
 
+You can enable the `pg_stat_statements` extension on your RDS instance by following these instructions:
+
+- Create a new custom DB Parameter Group, or modify an existing one.
+- Set the following settings:
+```
+shared_preload_libraries = pg_stat_statements (required)
+track_activity_query_size = 2048 (recommended if you have long queries)
+pg_stat_statements.track = ALL (track statements inside stored procedures)
```
Comment (Preetam) [inline content/getting-started/installing.md]: Little easier to read in this format?

```
- Set the following settings:
    - `shared_preload_libraries = pg_stat_statements` (required)
    - `track_activity_query_size = 2048` (recommended if you have long queries)
    - `pg_stat_statements.track = ALL` (track statements inside stored procedures)
```

- Set the following settings:
    - `shared_preload_libraries = pg_stat_statements` (required)
    - `track_activity_query_size = 2048` (recommended if you have long queries)
    - `pg_stat_statements.track = ALL` (track statements inside stored procedures)

**Comment by Preetam** [comment]
LGTM. :+1:

File: content/getting-started/installing.md (hunk 1 above)
Comment (potocnyj) [inline content/getting-started/installing.md]: Yeah, definitely looks better IMO, thanks for the tip!

**Pull request merged**